OWASPZAP.md

```diff
@@ -0,0 +1,141 @@
+# OWASP ZAP
+
+[OWASP ZAP](https://www.zaproxy.org/) is an open-source web application security scanner.
+
+## Description
+
+By enabling ZAP in the integration test, the [ZAP full scan GitHub action](https://github.com/marketplace/actions/owasp-zap-full-scan) will run a ZAP full scan that attacks the web application to find additional vulnerabilities.
+
+The alerts will be maintained as a GitHub issue in the corresponding repository as well as an artifact in the Integration Test workflow.
+
+## Warnings
+
+- The script can potentially run for a long period. The default value for ``zap-cmd-options`` prevents that by setting a max time of 60 minutes. Mind that when changing this parameter.
+- By deciding to run against an external target, since it simulates an attack, you should only use the full scan against targets that you have permission to test.
+
+## How to use
+
+If there is no need for customization, the test can be enabled by setting the parameter ``zap-dast-enabled`` to true.
+
+Then, after the integration tests, the ZAP full scan will run against the Charm unit IP address (port 80) and the logs can be viewed in the Job output, the artifact, or if vulnerabilities are found, in an issue entitled 'OWASP ZAP report'.
+
+If future scans identify a fixed issue or new alerts the action will update the issue with the required information.
+
+## Examples
+
+### Default
+
+```yaml
+jobs:
+  integration-tests:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    secrets: inherit
+    with:
+      zap-dast-enabled: true
```

Comment:
If it is default enabled, what target does it use for the test?

Reply:
it was only in the README.md file ("If this is not set, the unit IP address will be used as ZAP target"). I added to the OWASPZAP.md too now.